openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -160,14 +201,14 @@ def test_mutiple_program_engagement(self, mock_get_programs):
             ProgramFactory(
                 courses=[
                     CourseFactory(course_runs=[
-                        CourseRunFactory(key=newer_course_run_key),
+                        CourseRunFactory(key=newer_course_run_key, type='verified'),
```

Comment:
Are you sure passing type to `CourseRunFactory` is necessary? The factory is already providing the 'verified' type.